src/modules/RokulRunnings.ts

```diff
@@ -91,7 +91,7 @@ export class RokulRunnings {
   }
 
   /** Returns a list of installed channels as an array of objects */
-  getApps(retries = this.retries) {
+  async getApps(retries = this.retries) {
```

Comment:
these can return promises without being labelled as `async`.

Reply:
Removed and added a comment... I feel like this is not the first time I've tried to label these `async`

Reply:
Fixed in 9262563